tests/baselines/reference/jsdocParameterTagSnippetCompletion1.baseline

```diff
@@ -0,0 +1,15640 @@
+// | tutorial
+// | type
+// | typedef
+// | var
+// | variation
+// | version
+// | virtual
+// | yields
+// | param {*} param0 
+// | ----------------------------------------------------------------------
+//  */
+// function ee({ [someconst]: b }) {
+// }
+// /**
+//  * 
+//  * @p
+//      ^
+// | ----------------------------------------------------------------------
+// | abstract
+// | access
+// | alias
+// | argument
+// | async
+// | augments
+// | author
+// | borrows
+// | callback
+// | class
+// | classdesc
+// | constant
+// | constructor
+// | constructs
+// | copyright
+// | default
+// | deprecated
+// | description
+// | emits
+// | enum
+// | event
+// | example
+// | exports
+// | extends
+// | external
+// | field
+// | file
+// | fileoverview
+// | fires
+// | function
+// | generator
+// | global
+// | hideconstructor
+// | host
+// | ignore
+// | implements
+// | inheritdoc
+// | inner
+// | instance
+// | interface
+// | kind
+// | lends
+// | license
+// | link
+// | linkcode
+// | linkplain
+// | listens
+// | member
+// | memberof
+// | method
+// | mixes
+// | module
+// | name
+// | namespace
+// | overload
+// | override
+// | package
+// | param
+// | private
+// | prop
+// | property
+// | protected
+// | public
+// | readonly
+// | requires
+// | returns
+// | satisfies
+// | see
+// | since
+// | static
+// | summary
+// | template
+// | this
+// | throws
+// | todo
+// | tutorial
+// | type
+// | typedef
+// | var
+// | variation
+// | version
+// | virtual
+// | yields
+// | param {Object} param0 
+// | * @param {*} param0.a 
+// | ----------------------------------------------------------------------
+//  */
+// function ff({ "a": b }) {
+// }
+// /**
+//  * 
+//  * @p
+//      ^
+// | ----------------------------------------------------------------------
+// | abstract
+// | access
+// | alias
+// | argument
+// | async
+// | augments
+// | author
+// | borrows
+// | callback
+// | class
+// | classdesc
+// | constant
+// | constructor
+// | constructs
+// | copyright
+// | default
+// | deprecated
+// | description
+// | emits
+// | enum
+// | event
+// | example
+// | exports
+// | extends
+// | external
+// | field
+// | file
+// | fileoverview
+// | fires
+// | function
+// | generator
+// | global
+// | hideconstructor
+// | host
+// | ignore
+// | implements
+// | inheritdoc
+// | inner
+// | instance
+// | interface
+// | kind
+// | lends
+// | license
+// | link
+// | linkcode
+// | linkplain
+// | listens
+// | member
+// | memberof
+// | method
+// | mixes
+// | module
+// | name
+// | namespace
+// | overload
+// | override
+// | package
+// | param
+// | private
+// | prop
+// | property
+// | protected
+// | public
+// | readonly
+// | requires
+// | returns
+// | satisfies
+// | see
+// | since
+// | static
+// | summary
+// | template
+// | this
+// | throws
+// | todo
+// | tutorial
+// | type
+// | typedef
+// | var
+// | variation
+// | version
+// | virtual
+// | yields
+// | param {*} a 
```

Comment:
why isn't `param {Object} param0 ...` offered for `b` here?

Reply:
Because we'd be positionally annotating the first parameter with a completion at this position, not the second parameter, and `{ b }` is the second parameter. So inserting `@param {Object} param0 * @param {*} param0.b` here would not positionally match the `{ b }` parameter.

Reply:
ah, at first I didn't get the distinction in suggestions offered for positional vs named. Now I do. Thanks.